src/styles/tailwind/_base.scss

```
@@ -1821,6 +1821,726 @@
   background-color: get-theme(colors , red , 5);
 }
 
+.bp3-dark .dark-hover\:bg-inherit:hover {
+  background-color: inherit;
+}
+
+.bp3-dark .dark-hover\:bg-transparent:hover {
+  background-color: get-theme(colors , transparent);
```

```suggestion
  background-color: get-theme(colors, transparent);
```
Could you remove the whitespace before each comma? Thank you

---

Those whitespaces were generated by build process and were there originally in other classes. If I remove them, they would be overwritten within next build I suppose. Should I remove them anyway?

---

Ah gotcha. No, it's okay.
Just apply the feedback from Marc regarding shadow and we are good to go!